.github/project-labeler.yml

```diff
@@ -30,7 +30,7 @@ eslint-config-pie:
 - 'packages/tools/eslint-config-pie/**/*'
 
 stylelint-config-pie:
-- 'packages/tools/pie-stylelint-config/**/*'
+- 'packages/tools/stylelint-config-pie/**/*'
```

Comment:
lgtm, do we need to update any references in the monorepo?

Reply:
Nope – I did a find and replace and because the package name itself was changed before, all those links have been updated previously through that link. It's only this one left from what I could see

Reply:
Nice!